src/appleseed.python/bindshadergroup.cpp

```diff
@@ -121,9 +121,9 @@ namespace
             return m_shader_query->get_shader_type();
         }
 
-        size_t get_num_params() const
+        size_t get_param_count() const
         {
-            return m_shader_query->get_num_params();
+            return m_shader_query->get_param_count();
```

Comment:
You're right. That being said, I just renamed a method to conform to our coding conventions, and tried to keep the changes minimal.